### ISSUE TITLE
Minified LESS/CSS Output Changes

### ISSUE THREAD

**Issue by nissicreative** [bdkjones/CodeKit-1-Old#700]
Since updating to the latest CodeKit version, minified CSS files now concatenate multiple rules to a single, super-long line. This makes it very difficult to do diff checks between CSS files, and makes them pretty much human-unreadable. Furthermore, some—but not all—comments are preserved and appear in the minified output.

I'm assuming this has to do with a change to the LESS compiler? Is there any way to specify the old behavior, where each rule appears on its own line?

**Comment by bdkjones** [comment]
Yep. The new version of Less uses a better compressor, so that's the cause of the changes.

Any issues would need to be raised with the Less team directly.

**Comment by danieletesta** [comment]
@bdkjones so what's the difference **now** between Minified and Compressed?
Because to me it looks to be the same thing.
